zb-classifier.js

```diff
@@ -958,6 +958,29 @@ class ZigbeeClassifier {
     );
   }
 
+  addHumiditySensorProperty(node, msHumidityEndpoint) {
+    this.addProperty(
+      node,                           // device
+      'humidity',                     // name
+      {                               // property description
+        '@type': 'HumidityProperty',
+        label: 'Humidity',
+        type: 'number',
+        unit: 'percent',
+        minimum: 0,
+        maximum: 100,
+        readOnly: true,
+      },
+      PROFILE_ID.ZHA,                 // profileId
+      msHumidityEndpoint,             // endpoint
+      CLUSTER_ID.HUMIDITY,            // clusterId
+      'measuredValue',                // attr
+      '',                             // setAttrFromValue
+      'parsePercentAttr',             // parseValueFromAttr
+      CONFIG_REPORT_INTEGER
+    );
+  }
+
```

Comment:
In theory, this shouldn't be required since this classifier shouldn't be used with the xiaomi devices. Unfortunately, my device shows as lumi.weather, so I'll need to see if I can purchase one which reports as lumi.sensor_ht.

Reply:
On the other hand, isn't this adding humidity sensor support for generic sensors?
If so, would it make sense to open a separate PR for this?

Reply:
Yes - that would be adding humidity sensor support for generic sensors. I'd rather see that in a separate PR, since it isn't required for the xiaomi sensor which is what this PR is trying to support.

Reply:
Will do that next weekend, even though I don't have a generic zigbee temp/humidity sensor to test it on :confused: